tasks/main.yml

```diff
@@ -54,6 +54,8 @@
       {{ (tsdb_volume_type == "nfs") | ternary('--tsdb-nfs-path ' + tsdb_nfs_path,'') }} \
       {{ (tsdb_volume_type == "nfs") | ternary('--tsdb-nfs-host ' + tsdb_nfs_host,'') }} \
       {{ (tsdb_volume_type == "aws") | ternary('--tsdb-aws-volume-id ' + tsdb_aws_volume_id,'') }} \
+      {{ (apigw_cert is defined) | ternary('--apigw-cert ' + apigw_cert,'') }} \
+      {{ (apigw_key is defined) | ternary('--apigw-key ' + apigw_key,'') }} \
```

Comment:
The Key should be tied to the cert being passed.  So that we can bind the two as dependencies.
Similar to how we defined 'nfs' or smtp settings.
If you define a, apigw_cert you must define an apigw_key.

Reply:
I have made the change so that `apigw_cert` and `apigw_key` both are defined to set these flags.